design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -0,0 +1,261 @@
+## Gift Card product type support will be added after the Product Implementation is developed
+type Query {
+    "The requisitionList query returns the contents of customer's  Requisition List"
+    requisitionList(
+        id: ID!
+    ): RequisitionList
+
+    "Export Requisition list"
+    exportRequisitionList(
+        "unqiue Id of Requisition list"
+        id: ID!
+    ): CsvOutput
+}
+
+type Customer {
+    "Get Requisition Lists of customer"
+    requisitionLists(
+        pageSize: Int = 20,
+        currentPage: Int = 1
+    ): RequisitionLists
+}
+
+"Provides Requisition Lists of customer"
+type RequisitionLists  {
+    "List of Requisition Lists"
+    items: [RequisitionList]
+    "Page Information for pagination"
+    page_info: PageInfo
+    "Total count of Requisition Lists"
+    total_count: Int
+}
+
+"Requisition List Type"
+type RequisitionList {
+    "Unique Identifier of Requisition List"
+    id: ID!
+    "Name of the list"
+    name: String!
+    "Description of the list"
+    description: String
+    "Items in the list"
+    items: [RequisitionListItemInterface]
+    "Number of items in list"
+    items_count: Int!
+    "Latest Activity"
+    updated_at: String
+}
+
+"Interface type for Requisition List Item"
+interface RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+}
+
+type SimpleRequisitionListItem implements RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+    "custom Option selected"
+    customizable_options: [SelectedCustomizableOption]
+}
+
+type VirtualRequisitionListItem implements RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
```

Comment:
For cases like this where the description is identical to the name of the field, we can just remove the description